### DISCUSSION TITLE
Hi GLEAN training!

### DISCUSSION THREAD

**Discussion by scj0709** [open-mmlab/mmediting#1207] [How-to] [answered]
HI! I am impressed with your paper.

I am training GLEAN.
I want to make the learning rate of encoder and decoder different, but how do I do it?
Thanks~

**Comment by ckkelvinchan** [comment, marked as answer]
You can use the argument `paramwise_cfg` to set the learning rate based on keywords. 

An example is shown [here](https://github.com/open-mmlab/mmediting/blob/master/configs/restorers/basicvsr/basicvsr_reds4.py#L124)
